packages/react/src/store/userStore.js

```diff
@@ -38,6 +38,8 @@ const useUserStore = create((set) => ({
     set(() => ({ showCurrentUserInfo })),
   currentUser: {},
   setCurrentUser: (currentUser) => set({ currentUser }),
+  dark: false,
```

Comment:
why is this required?

Reply:
Not required here ! 
Sorry i forgot to remove it from userStore. I already created this in theme store

Reply:
Removed it @sidmohanty11